src/BigAssApi.js

```diff
@@ -0,0 +1,94 @@
+const dgram = require("dgram");
+const BigAssFan = require('./BigAssFan');
+const BigAssLight = require('./BigAssLight');
+const BigAssProperty = require('./BigAssProperty');
+const { syncingCallback, retryCall, myLogWrapper } = require('./utils');
+
+function FanMaster (numberOfExpectedDevices) {
+    this.allDevices = {}; // Dictionary of fan name -> BigAssDevice
```

Comment:
This change should be in README.md - and might impact Homekit support, so I'm not sure how I feel about the renaming purely from a backwards compatibility perspective